server/src/app.controller.spec.ts

```diff
@@ -14,6 +14,7 @@ describe('AppController', () => {
     appController = app.get<AppController>(AppController);
   });
 
+  // i'm not sure what this is:
```

Comment:
good call, our tests are woefully sad
putting in an issue now
https://github.com/Nonprofit-Exchange-Hub/web-app/pull/78
this PR will skip this pointless test

it was mocking up our app controller and then hitting an endpoint called "root" (that no longer exists) and expecting it to return that string. happy to talk through it more if still unclear

Reply:
don't forget to remove this comment before merging

Reply:
Lol. Sorry! I saw the TS error and meant to investigate and then forgot.  Makes sense though for sure.